### PR TITLE
Fix #22: Added Chirpstack provisioning script and updated README

### LINKED ISSUE #22: Add Chirpstack provisioning script and update readme

### PR DESCRIPTION
**Pull request by sivaprakash-mcci**

### CODE REVIEW

File: mcci_catena_provision_chirpstack.py [hunk 1]
```diff
@@ -0,0 +1,1280 @@
...
+        oAppContext.error('Port {} already closed'.format(sPortName))
+        return None
+
+##############################################################################
+#
+#   main 
+#
+##############################################################################
+
+if __name__ == '__main__':
```
Comment (terrillmoore) [inline mcci_catena_provision_chirpstack.py]: please make a separate main function:

```python
def main():
   # body of main
   ''' all the code from lines 1094 to 1280 goes here

# standard wrapper
if __name__ == '__main__':
   main()
```

That way the main function is not special. You'll probably need to add `global` declaration for things like `pName`, `pDir`, etc.

File: mcci_catena_provision_chirpstack.py [hunk 2]
```diff
@@ -0,0 +1,1280 @@
...
+        help='Warning messages become error messages')
+    optparser.add_argument(
+        '-s',
+        action='store',
+        nargs=1,
+        dest='script',
+        type=str,
+        help='Specify script name to load catena info')
+
+    opt = optparser.parse_args()
```
Comment (terrillmoore) [inline mcci_catena_provision_chirpstack.py]: Please move this to a separate function, called at this point.  I usually call it `ParseArguments()`. The function should return the result of `optparser.parse_args()`.  This is for clarity.

**Review by terrillmoore — APPROVED** [review]
Looks very good. Comments are cosmetic, not functional.

File: mcci_catena_provision_chirpstack.py (hunk 1 above)
Comment (sivaprakash-mcci) [reply mcci_catena_provision_chirpstack.py]: As per the suggestion, created separate main function

File: mcci_catena_provision_chirpstack.py (hunk 2 above)
Comment (sivaprakash-mcci) [reply mcci_catena_provision_chirpstack.py]: Moved the argparser code to the separate function `parse_arguments()`

**Review by terrillmoore — APPROVED** [review]

**Pull request merged**